src/amo/sagas/home.js

```diff
@@ -32,20 +29,22 @@ export function* fetchHomeAddons({
   try {
     const state = yield select(getState);
 
+    const collections = [];
+    // eslint-disable-next-line no-restricted-syntax
+    for (const coll of collectionsToFetch) {
```

Comment:
Can we use a more descriptive variable name for this over `coll`? I think `collection` is fine.

Also, if we're going to keep using `for... of` we shouldn't have it be an eslint violation. I'm fine with doing it, but it should be an okay thing to do 😄

Reply:
I tried to use `forEach` and `map` instead of `for... of`, based on the conversation in an earlier issue, but couldn't seem to figure out the syntax that would work because of the `yield` and generators. If we're okay with using `for... of` in some situations (which seems to be what you're saying here), I agree we should fix the eslint rule. @tofumatt Is it okay to just add that to this PR and commit, or would you prefer that a separate issue and commit be done for that?

Reply:
Ah, I see that the eslint config doesn't even reside in this repo, so I guess it would have to be a separate commit.